pkg/endpoints/cluster.go

```diff
@@ -162,3 +164,70 @@ func (r Resource) GetProperties(request *restful.Request, response *restful.Resp
 	properties := Properties{InstallNamespace: os.Getenv("INSTALLED_NAMESPACE")}
 	response.WriteEntity(properties)
 }
+
+// Get dashboard version
+func (r Resource) GetDashboardVersion(request *restful.Request, response *restful.Response) {
+	config, err := rest.InClusterConfig()
```

Comment:
Surely we've got one of these we should be reusing instead of creating new ones in both methods

Reply:
I have removed them in both functions and now am using the K8sClient one instead and combined it with getting the deployments